### PR TITLE
Feat: UI Refresh of the Press Kit

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
UI refresh of the Press Kit
Preview: https://deploy-preview-67--opensauced-landing.netlify.app/press

## Mobile & Desktop Screenshots/Recordings
Screenshot 2022-10-09 at 6 36 15 PM

Screenshot 2022-10-09 at 6 36 23 PM

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
Can we add a max width on the images? image

File: components/Footer2.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            </div>
+            <div className="grow">
+                <div className="font-bold mb-2 text-gray-600 text-xs">MENU</div>
+                <ul className="text-sm text-gray-400">
+                    <li><Link href="/blog">Blog</Link></li>
+                    <li><Link href="/press">Press Kit</Link></li>
+                </ul>
+            </div>
+            <div className="grow">
+                <div className="font-bold mb-2 text-gray-600 text-xs">MORE SAUCE</div>
```
Comment (bdougie) [inline components/Footer2.tsx:44]:
```suggestion
                <div className="font-bold mb-2 text-gray-600 text-xs">ADDITIONAL RESOURCES</div>
```

File: components/Footer2.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                <ul className="text-sm text-gray-400">
+                    <li><a href="https://hot.opensauced.pizza">hot.opensauced.pizza</a></li>
+                    <li><a href="https://insights.opensauced.pizza">insights.opensauced.pizza</a></li>
+                </ul>
+            </div>
+            <div className="grow">
+                <div className="font-bold mb-2 text-gray-600 text-xs">CONTACT</div>
+                <div className="text-sm text-gray-400">
+                    +1(123) 456-7890
```
Comment (bdougie) [inline components/Footer2.tsx:53]: This should be hello@opensauced.pizza and be a link to email

File: components/Footer2.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                </div>
+            </div>
+        </div>
+        <div className="grow flex justify-between mt-10 py-6 border-t-2 border-gray-700">
+            <div className="text-sm text-gray-500">C 2022 Open Sauced, LLC. All rights reserved.</div>
+            <div className="text-sm text-gray-500">
+                <ul className="flex gap-2">
+                    <li>Privacy</li>
+                    <li>Terms</li>
+                    <li>Status</li>
```
Comment (bdougie) [inline components/Footer2.tsx:63]: These should all be links. Please link with the same links from the landing page

**Review by bdougie — CHANGES_REQUESTED** [review]
Some of the links to need be updated

**Comment by shamimbinnur** [comment]
@bdougie I am not sure why are you seeing like that, while I am seeing this! But I will figure it out !

**Comment by bdougie** [comment]
I am on an iPhone. I'll check with about her iPhone user to confirm.

Previous image was mobile safari on iOS and this image is mobile chrome on iOS 

image

**Comment by shamimbinnur** [comment]
> I am on an iPhone. I'll check with about her iPhone user to confirm.
> 
> Previous image was mobile safari on iOS and this image is mobile chrome on iOS
> 
> image

@bdougie hopefully, this is fixed now!

**Review by bdougie — APPROVED** [review]

File: components/Footer2.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                <div className="font-bold mb-2 text-gray-600 text-xs">ADDITIONAL RESOURCES</div>
+                <ul className="text-sm text-gray-400">
+                    <li><a href="https://hot.opensauced.pizza">hot.opensauced.pizza</a></li>
+                    <li><a href="https://insights.opensauced.pizza">insights.opensauced.pizza</a></li>
+                </ul>
+            </div>
+            <div className="grow">
+                <div className="font-bold mb-2 text-gray-600 text-xs">CONTACT</div>
+                <div className="text-sm text-gray-400">
+                    <p>hello@opensauced.pizza</p>
```
Comment (bdougie) [inline components/Footer2.tsx:53]:
```suggestion
                    <p><a href="mailto:hello@opensauced.pizza">hello@opensauced.pizza</a></p>
```

File: components/Footer2.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                        <BsDiscord className="text-white text-2xl" />
+                    </a>
+                    <a href="https://dev.to/opensauced">
+                        <FaDev className="text-white text-2xl" />
+                    </a>
+                </div>
+
+            </div>
+            <div className="grow">
+                <div className="font-bold mb-2 text-gray-600 text-xs">MENU</div>
```
Comment (bdougie) [inline components/Footer2.tsx:37]:
```suggestion
                <div className="font-bold mb-2 text-gray-600 text-xs">LINKS</div>
```

**Pull request merged**